docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,40 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying and cutting of blocks from one page, and pasting them to another page.
+This chapter describes how to do so.
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold `ctrl` key to select multiple blocks you want to copy/cut.
+Once blocks are selected, you will have a copy <img alt="../_static/copy.svg" src="../_static/copy.svg" style="height: 1em;"> and a cut <img alt="../_static/cut.svg" src="../_static/cut.svg" style="height: 1em;"> option at the bottom of Toolbar.
```

Comment:
sphinx-book-theme has an additional style that we want to override for inline images.

```css
img {
    margin:1rem 0;
}
```

So let's add a custom class.

```css
/* override sphinx-book-theme img tag with a class */
img.inline {
    margin: 0;
    height: 1em;
}
```

If that does not work, then add ` !important` to the styles as needed.

Also modify the inline HTML.

```suggestion
Once blocks are selected, you will have a copy <img alt="../_static/copy.svg" src="../_static/copy.svg" class="inline"> and a cut <img alt="../_static/cut.svg" src="../_static/cut.svg" class="inline"> option at the bottom of Toolbar.
```

Reply:
I should add, I don't know if that MyST extension is doing what it promises. Let's make sure it does in fact add ` class="inline"` to the `<img>` tag when it is rendered.